papas/pfobjects.py

```diff
@@ -279,7 +279,17 @@ def info(self):
     def short_info(self):
         return '{e:.1f}'.format(
             e = self.energy,
-        )     
+        )
+
+    def __repr__(self):
```

Comment:
why adding these methods? especially not necessary since you have also added these methods in the base class